### DISCUSSION TITLE
Studio-Elf Toast 1.0.0 is Released

### DISCUSSION THREAD

**Discussion by leigh-pointer** [oqtane/oqtane.framework#3395] [Show and tell]
Today 18-Oct-2023 Studio-Elf Toast was released to the Oqtane Commercial Market place [ https://www.oqtane.net/ ].  The module comes with a 15 day trial license. Feel free to give it a whirl.

![icon](https://github.com/oqtane/oqtane.framework/assets/9447612/9b35185c-5816-490c-ba68-3b63c3c63f92)

Check it out https://www.studio-elf.net/oqtane/toast/

**Comment by thabaum** [comment]
Congrats! I was hoping to see a fire light up here!

Cheers!
